packages/webapp/pages/onboarding.tsx

```diff
@@ -73,6 +73,17 @@ import { LoggedUser } from '@dailydotdev/shared/src/lib/user';
 import { defaultOpenGraph, defaultSeo } from '../next-seo';
 import styles from '../components/layouts/Onboarding/index.module.css';
 
+type OnboardingVisual = {
+  showCompanies?: boolean;
+  fullBackground?:
+    | {
+        mobile?: string;
+        desktop?: string;
+      }
+    | false;
+  image?: string;
+};
```

Comment:
Totally fine like this as well, just another suggestion that makes more sense in my opinion, but no strong opinion. Only change if it makes sense for u as well.
```suggestion
type OnboardingVisual = {
  showCompanies?: boolean;
  fullBackground?: boolean;
  image?: string;
  imageMobile?:string
};
```

Reply:
I think it's fine to keep it in `fullBackground`. It defaults to false (or undefined if not specified)